verilog/formatting/formatter.cc

```diff
@@ -84,6 +84,7 @@ static void DeterminePartitionExpansion(partition_node_type* node,
   const auto partition_policy = uwline.PartitionPolicy();
   VLOG(3) << "partition policy: " << partition_policy;
   switch (partition_policy) {
+    case PartitionPolicyEnum::kAppendFittingSubpartitions:
```

Comment:
(I understand this is a placeholder for the moment.)

Reply:
Yes it was. Fixed.